packages/adblocker-webextension/adblocker.ts

```diff
@@ -135,12 +136,13 @@ export function filterRequestHTML(
   filterResponseData: Browser['webRequest']['filterResponseData'],
   { id }: { id: string },
   rules: HTMLSelector[],
+  modifiers: HTMLModifier[],
```

Comment:
Would it make sense to unify these two and convert `HTMLSelector` into HTMLModifier? Or are they too different semantically?

Reply:
`HTMLSelector` is optimised to be used in streaming but `HTMLModifier` isn't. The former one is expectable but the later one is not in the current implementation.

Reply:
This will be handled with a separate PR as an improvement.